indico/htdocs/js/indico/jquery/global.js

```diff
@@ -165,7 +165,7 @@ $(document).ready(function() {
         }
     });
 
-    $('.js-dropdown').each(function() {
+    $('body').on('mouseenter', '.js-dropdown', function() {
```

Comment:
not sure if that's a good idea. that event is going to be triggered like crazy (whenever you hover an element). maybe it'd be better to explicitly call the dropdown js on dynamically created elements when necessary